app/controllers/core_induction_programmes/mentor_materials_controller.rb

```diff
@@ -1,11 +1,13 @@
 # frozen_string_literal: true
 
-class CoreInductionProgrammes::MentorMaterialsController < ApplicationController
+class CoreInductionProgrammes::MentorMaterialsController < CoreInductionProgrammes::LessonsController
```

Comment:
Are we doing all of this controller inheritance to have access to `load_xxx` methods that are shared between them?

If so, I would really prefer those load methods to be in a helper, and include that helper in all relevant controllers. That saves us the need to worry about what actions we need to skip, and what needless methods we inherit.